### ISSUE TITLE
No timeout parameter in requests.get() call

### ISSUE THREAD

**Issue by ayy1337** [ericsomdahl/python-bittrex#132]
```def using_requests(request_url, apisign):
    return requests.get(
        request_url,
        headers={"apisign": apisign}
    ).json()```

See pic..
![screenshot_2018-01-25_05-08-17](https://user-images.githubusercontent.com/24858633/35349002-de5b3f68-018d-11e8-8a53-9944721af999.jpg)
http://docs.python-requests.org/en/master/user/quickstart/#timeouts

**Comment by ericsomdahl** [comment]
that is a reasonable observation.  Please submit a pull request

**Comment by ayy1337** [comment]
Unit test failed on 2.7 version, not quite sure why though as it worked for me locally..

**Comment by ericsomdahl** [comment]
addressed in PR #133
